packages/ui-components/src/components/AppShell/AppShell.component.tsx

```diff
@@ -55,22 +56,23 @@ export const AppShell = ({
         </>
       ) : (
         <>
-          {pageHeader && (typeof pageHeader === "string" || pageHeader instanceof String) ? (
-            <PageHeader heading={pageHeader} />
-          ) : (
-            pageHeader
-          )}
-          {topNavigation && topNavigation}
-          {/* Wrap everything except page header and footer and navigations in a main container. Add top margin to MainContainerInner as we are not in embedded mode here. */}
+          <HeaderContainer fullWidth={fullWidthContent === true ? true : false} className="">
```

Comment:
Please change `{fullWidthContent === true ? true : false}` to just `{fullWidthContent === true}` for simplicity